express-post-json/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+
+const app = express();
+
+
+let count = 0
+const grades = [
+
+];
+express.json()
+app.use(express.json(grades))
```

Comment:
The argument for `express.json()` is some optional options in an object, so you don't need to pass your data array in.